chain/messagepool/messagepool.go

```diff
@@ -313,7 +315,7 @@ func (mp *MessagePool) addTs(m *types.SignedMessage, curTs *types.TipSet) error
 
 	snonce, err := mp.getStateNonce(m.Message.From, curTs)
 	if err != nil {
-		return xerrors.Errorf("failed to look up actor state nonce: %w", err)
+		return xerrors.Errorf("failed to look up actor state nonce: %w: %w", err, ErrBroadcastAnyway)
```

Comment:
nitpick: I don't think this supports more than one %w verb, probably want to make the first one just a normal %s

Reply:
For the future, the "most" correct for errors is `%v` (unless it is the last one wrapping then `%w`.